front/app/containers/UsersShowPage/Following/index.tsx

```diff
@@ -66,7 +66,7 @@ const Following = () => {
   trackEventByName(tracks.browseFollowsInActivityPage);
 
   return (
-    <Box display="flex" w="100%" flexDirection="column">
+    <Box display="flex" w="100%" flexDirection="column" id="tab-following">
```

Comment:
An accessibility issue discovered - the tabs were not associated with the tab panels.